src/App.css

```diff
@@ -1,3 +1,64 @@
 .App {
-  text-align: center;
+  width: 100%;
+  height: 100vh;
+}
+
+:root {
```

Comment:
what this selector dose?
also, what `--containersOpacity` dose?

Reply:
It a CSS var (:
I use it so if i want to show the containers for the buttons i change it to 0.3 or 1 to make all the containera opace in 1 change.

Reply:
reffered to in person.